tools/src/main/scala/scala/scalanative/build/LLVM.scala

```diff
@@ -55,8 +55,10 @@ private[scalanative] object LLVM {
           if (config.targetsWindows) Seq("-g")
           else Nil
         }
+        val expectionsHandling =
```

Comment:
Little late but this should have been `exceptionsHandling`.

Reply:
I fixed this in https://github.com/scala-native/scala-native/pull/2572 :)